barretenberg/cpp/src/barretenberg/flavor/ultra.hpp

```diff
@@ -200,7 +225,7 @@ class Ultra {
         // Gemini-specific getters.
         RefVector<DataType> get_unshifted()
         {
-            return { q_c,           q_l,   q_r,      q_o,     q_4,     q_m,          q_arith, q_sort,
+            return { q_m,           q_c,   q_l,      q_r,     q_o,     q_4,          q_arith, q_sort,
```

Comment:
err, makes me nervous. Did this have to match some other ordering?

Reply:
This was a bug fix that was hard to find - this is matching the order in PrecomputedEntities, which makes up proving key, but not AllEntities. AllEntities should likely be refactored to be the inheritance of PrecomputedEntities + WitnessEntities

Reply:
I can take that up if you like